.github/actions/check-issues-are-disabled/action.yml

```
@@ -0,0 +1,24 @@
+name: Check Issues Are Disabled
+description: Checks whether the current repository has disabled the issues feature.
+
+outputs:
+  issues-disabled:
+    description: Returns a boolean value indicating whether the issues feature is disabled.
+    value: ${{ steps.repository-has-disabled-issues.outputs.issues-disabled }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Scripts
+      shell: pwsh
+      run: |
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+        (Resolve-Path "${{ github.action_path }}/../../../Scripts").Path >> $Env:GITHUB_PATH
+  
+    - name: Check Issues Are Disabled In Current Repository
```

```suggestion
    - name: Check Issues are Disabled in Current Repository
```
To make it consistent with other titles.